### PR TITLE
member page

### PR DESCRIPTION
**Pull request by joelmayemba**
Add dev members

### CODE REVIEW

**Review by chouaibj — APPROVED** [review]

File: src/Model/MembersManager.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Model;
+
+class MembersManager extends AbstractManager
+{
+    public const TABLE = 'MEMBERS';
+
+    public function selectAllMembers(): array
```
Comment (Benja-san) [inline src/Model/MembersManager.php:9]: if your method only call the parent method selectAll, you can suppress it and directly use the selectAll method when needed in controllers

File: src/View/Members/index.html.twig
Comment (Benja-san) [inline src/View/Members/index.html.twig:1]: No H1 on this page

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/View/Members/index.html.twig
Comment (joelmayemba) [reply src/View/Members/index.html.twig:1]: Done

File: src/Model/MembersManager.php (hunk 1 above)
Comment (joelmayemba) [reply src/Model/MembersManager.php:9]: Done

**Review by Benja-san — CHANGES_REQUESTED** [review]
Remove all useless methods in managers (selectAllMembers is not used anymore) (getMocktailById only referes to its parent method so it can be erased in favor of selectOneById from abstract model) (and so on...)

**Review by Benja-san — APPROVED** [review]

**Pull request merged**